cob_hardware_config/cob4-2/config/light_torso.yaml

```diff
@@ -2,6 +2,7 @@ devicestring: /dev/ttyLED
 devicedriver: stageprofi
 baudrate: 38400
 num_leds: 58
+led_offset: 5
```

Comment:
Is there a default value `0`?
Do the light yamls for the other robots need this new parameter, too?